_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -49,32 +49,31 @@ Testing that the [accessible name][] describes the purpose of the `object` eleme
 
 #### Passed Example 1
 
-This `object` element has a non-empty [accessible name][] through its `aria-label` attribute.
+This `object` element which embeds an audio resource has a non-empty [accessible name][] through its `aria-label` attribute.
 
 ```html
 <object aria-label="Moon speech" data="/test-assets/moon-audio/moon-speech.mp3"></object>
 ```
 
 #### Passed Example 2
 
-This `object` element has a non-empty [accessible name][] through its `title` attribute.
+This `object` element which embeds a video resource has a non-empty [accessible name][] through its `title` attribute.
 
 ```html
-<object title="Moon speech" data="/test-assets/moon-audio/moon-speech.mp3"></object>
+<object title="My University" data="/test-assets/rabbit-video/video.mp4"></object>
```

Comment:
The title on this doesn't seem appropriate for the video.

```suggestion
<object title="Rabbit animated short" data="/test-assets/rabbit-video/video.mp4"></object>
```

Reply:
Good catch